README.md

```diff
@@ -1,8 +1,57 @@
+# Tina Lee . Jarvis Consulting
 
+I am a new graduate from the University of Toronto with a Bachelor's degree specializing in Bioinformatics. During my university years, I had opportunities to work with professional teams as a research student to build a solid programming foundation and explore my interest in data analysis. Besides, I am also a team player and an effective communicator, who is always responsible and well-organized. Currently, I am working at  Jarvis to improve my skill sets as a data engineer while building challenging projects and applications using new technologies such as Docker, Springboot, Java, Maven, REST API, RDBMS, etc. I would say I am a problem-solver by nature and I am ready to solve challenging problems in our daily lives in the data engineer industry.
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git, SQL, IntelliJ, Maven
+
+**Competent:** Docker, JUnit/Mockito, REST API, Spring Boot, MVC architecture
+
+**Familiar:** Javascript, React Native, R, Python, Express.js
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_TinaLee](https://github.com/jarviscanada/jarvis_data_eng_TinaLee)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_TinaLee/tree/master/linux_sql)]: Implemented a Cluster Monitoring Agent that aids Jarvis Linux Cluster Administration (LCA) team in future resource planning. This project is implemented using bash shell scripts to automatically collect hardware specification data and resource usage data of each server. Data will be collected and stored to a Postgres SQL generated by Docker every one-minute using crontab.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_TinaLee/tree/master/core_java)]:
+      
+  - Twitter App: Designed a Twitter application with MVC architecture that allows users to perform CRUD functionalities via REST API. Programmed using Twitter API, HTTP client, OAuth authorization, and JSON serialization libraries.
+  - JDBC App: Implemented a simple application that sets up a connection between a Java application and RDBMS for data accessing. The CRUD functionalities are implemented using DAO design patterns.
+  - Grep App: Developed a Java application that mimics the grep command line function. The app recursively searches for a matched pattern given a root directory and outputs matched lines to an output file.
+
+**Springboot App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_TinaLee/tree/master/springboot)]: Designed a Java application that simulates an online stock trading platform using microservice and MVC architecture. The application fetches the latest market data from IEX Cloud and stores this information along with trader information in the PostgreSQL database. Users could buy or sell stocks and deposit or withdraw funds from their accounts. This application is managed and packaged by Maven and Spring Boot.
```

Comment:
Designed and developed a REST API that .... using Java 8, Springboot, microservice ....

again, missing testing and deployment